src/app/beer_garden/requests.py

```diff
@@ -1052,6 +1027,7 @@ def handle_event(event):
 
                     if getattr(existing_request, field) != new_value:
                         request_changed[field] = new_value
+                        setattr(existing_request, field, new_value)
 
                 # Add output fields only if the status changes to a compelted state
                 if "status" in request_changed:
```

Comment:
This line is why I did not change request_changed to a boolean.

Reply:
yes, and if it is set to `True` then it will execute. The same as if the dictionary was populated.

Reply:
```if "status" in True``` will throw an error. It needs to either be a dict or a list to work like this.

Reply:
I concede